packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -287,4 +288,49 @@ describeSpec('Limbo Documents:', [], () => {
         .expectEvents(query, { removed: [doc2] });
     }
   );
+
+  specTest(
+    'Limbo documents are resolved after primary tab failover',
+    ['multi-client'],
+    () => {
+      const query = Query.atPath(path('collection'));
+      const doc1 = doc('collection/a', 1000, { key: 'a' });
+      const doc2 = doc('collection/b', 1001, { key: 'b' });
```

Comment:
nit: I like using docA / docB (to match the actual document names).

Reply:
Fixed.